tests/VolleyManagement.Domain.UnitTests/TeamTests.cs

```diff
@@ -205,15 +205,171 @@ public void Team_RosterContainsAlreadyTheSamePlayerId_ExceptionIsThrown()
             //Act
             Action act = () =>
             {
-                team.Roster = new List<PlayerId>() {
-            new PlayerId(2),
-            new PlayerId(2)
-                    };
+                team.AddPlayers(new List<PlayerId>() {
```

Comment:
How this differs from the next test?

Reply:
this check when we add few the same Player Id, next check if we add PlayerID that already in Roster